### ISSUE TITLE
What does the plot 'batch_join/fraction_of_4096_full' mean?

### ISSUE THREAD

**Issue by uzair789** [davidsandberg/facenet#380]
Hi, When I run training, and visualize the progress on tensorboard, I see a plot named  'batch_join/fraction_of_4096_full' . What exactly is this plot representing? I went through the code multiple times searching where the summary for this plot was being written but I couldn't find it. Is this plot being implicitly created by the batch_join function?

**Comment by davidsandberg** [comment]
Hi,
It shows you how full the queue is. I can find the following paragraph in the [tensorflow tutorial](https://www.tensorflow.org/programmers_guide/reading_data) on the input pipeline:
"How many threads do you need? the tf.train.shuffle_batch* functions add a summary to the graph that indicates how full the example queue is. If you have enough reading threads, that summary will stay above zero. You can view your summaries as training progresses using TensorBoard."
and I guess that holds also for the non-shuffling batch_join.
